e2e/test/TelemetryPropertyNames.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    /// <summary>
+    /// The property names to log to telemetry.
+    /// </summary>
+    public class LoggerPropertyNames
```

Comment:
Isn't this a duplicate of Logging PropertyNames?

Reply:
I thought I renamed this. Not sure why they both exist. Let me take a look. It was renamed from TelemetryPropertyName -> LoggerPropertyNames.